python/hail/utils/__init__.py

```diff
@@ -0,0 +1,17 @@
+from .summary import Summary
+from .misc import FunctionDocumentation, wrap_to_list, get_env_or_default
+from .hadoop_utils import hadoop_copy, hadoop_read, hadoop_write
+from .struct import Struct
+from .java import error, warn, info
+
+__all__ = ['Summary',
```

Comment:
Should Summary be in hail.genetics?

Reply:
summary should be kaputt. We can fix it in a future PR